game_eggs/steamcmd_servers/hurtworld/egg-hurtworld.json

```diff
@@ -1,32 +1,32 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1",
+        "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2022-01-20T12:55:06-05:00",
+    "exported_at": "2023-10-07T13:37:51+08:00",
     "name": "Hurtworld",
     "author": "brycea@terrahost.cloud",
     "description": "Hurtworld is a hardcore multiplayer survival FPS with a focus on deep survival progression that doesn't become trivial once you establish some basic needs. Built for hardcore gamers, Hurtworld aims to punish.",
     "features": [
         "steam_disk_space"
     ],
-    "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_source"
-    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/games:source": "ghcr.io\/parkervcp\/games:source"
+    },
     "file_denylist": [],
-    "startup": ".\/Hurtworld.x86_64 -batchmode -nographics -exec \"host {{SERVER_PORT}};queryport {{QUERY_PORT}};maxplayers {{MAX_PLAYERS}};servername {{HOSTNAME}};creativemode ${CREATIVE_MODE};${ADMINS}\" -logfile $1",
+    "startup": ".\/Hurtworld.x86_64 -batchmode -nographics -exec \"host {{SERVER_PORT}};queryport {{QUERY_PORT}};maxplayers {{MAX_PLAYERS}};servername {{HOSTNAME}};addadmin {{ADMINS}}\" -logfile $1",
```

Comment:
Where did the creativemode mode whent to ? as the variable is still there

Reply:
> Where did the creativemode mode whent to ? as the variable is still there

Forgive me, I didn't see any variable settings about CREATIVE_MODE and ADMINS, so I changed it following the official launch command